seml/__main__.py

```diff
@@ -0,0 +1,678 @@
+import functools
+import json
+import logging
+import sys
+from pathlib import Path
+from typing import List
+
+from typing_extensions import Annotated
+
+import seml.typer as typer
+from seml.add import add_config_files
+from seml.configure import configure
+from seml.database import (clean_unreferenced_artifacts,
+                           get_collections_from_mongo_shell_or_pymongo,
+                           get_mongodb_config, list_database)
+from seml.manage import (cancel_experiments, delete_experiments, detect_killed,
+                         print_fail_trace, reload_sources, report_status,
+                         reset_experiments)
+from seml.settings import SETTINGS
+from seml.start import print_command, start_experiments, start_jupyter_job
+from seml.utils import LoggingFormatter, cache_to_disk
+
+States = SETTINGS.STATES
+
+
+@cache_to_disk('db_config', SETTINGS.AUTOCOMPLETE_CACHE_ALIVE_TIME)
+def db_collection_completer():
+    """ CLI completion for db collections. """
+    config = get_mongodb_config()
+    return list(get_collections_from_mongo_shell_or_pymongo(**config))
+
+
+app = typer.Typer(no_args_is_help=True)
+YesAnnotation = Annotated[bool, typer.Option(
+    '-y',
+    '--yes',
+    help="Automatically confirm all dialogues with yes.",
+)]
+SacredIdAnnotation = Annotated[int, typer.Option(
+    '-id',
+    '--sacred-id',
+    help="Sacred ID (_id in the database collection) of the experiment. "
+            "Takes precedence over other filters.",
+)]
+FilterDictAnnotation = Annotated[dict, typer.Option(
+    '-f',
+    '--filter-dict',
+    help="Dictionary (passed as a string, e.g. '{\"config.dataset\": \"cora_ml\"}') to filter "
+            "the experiments by.",
+    metavar='JSON',
+    parser=json.loads,
+)]
+BatchIdAnnotation = Annotated[int, typer.Option(
+    '-b',
+    '--batch-id',
+    help="Batch ID (batch_id in the database collection) of the experiments. "
+            "Experiments that were staged together have the same batch_id.",
+)]
+
+_STATE_LIST = [s for states in States.values() for s in states]
+FilterStatesAnnotation = Annotated[List[str], typer.Option(
+    '-s',
+    '--filter-states',
+    help='List of states to filter the experiments by. If empty (""), all states are considered.',
+    metavar=f'[{"|".join(_STATE_LIST)}]',
+    parser=lambda s: s.strip().upper(),
+    callback=lambda values: [
+        __x.strip().upper()
+        for _x in values
+        for __x in _x.replace(',', ' ').split()
+        if __x
+    ],
+)]
+SBatchOptionsAnnotation = Annotated[dict, typer.Option(
+    '-sb',
+    '--sbatch-options',
+    help="Dictionary (passed as a string, e.g. '{\"gres\": \"gpu:2\"}') to request two GPUs.",
+    metavar='JSON',
+    parser=json.loads
+)]
+NumExperimentsAnnotation = Annotated[int, typer.Option(
+    '-n',
+    '--num-experiments',
+    help="Number of experiments to start. "
+            "0: all (staged) experiments ",
+)]
+FileOutputAnnotation = Annotated[
```

Comment:
This previously was `--no-file-output` or `-nf`. Is there a way to recover these shorthands (to stay consistent with previous releases of seml)